packages/block-editor/src/components/rich-text/content.scss

```diff
@@ -28,7 +22,7 @@
 
 // Captions may have lighter (gray) text, or be shown on a range of different background luminosites.
 // To ensure legibility, we increase the default placeholder opacity to ensure contrast.
-figcaption.block-editor-rich-text__editable [data-rich-text-placeholder]::before {
```

Comment:
This was actually broken, so maybe I should just remove it. It was using ::before...